tests/core/mempool/test_mempool_manager.py

```diff
@@ -251,5 +256,36 @@ async def get_coin_record(_: bytes32) -> Optional[CoinRecord]:
 
     mempool_manager = await instantiate_mempool_manager(get_coin_record)
     conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1]]
-    _, _, result = await generate_and_add_spendbundle(mempool_manager, conditions)
+    _, _, result = await generate_and_add_spendbundle(mempool_manager, TEST_COIN, conditions)
     assert result == (None, MempoolInclusionStatus.FAILED, Err.UNKNOWN_UNSPENT)
+
+
+@pytest.mark.asyncio
+async def test_same_sb_twice_with_eligible_coin() -> None:
+    async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
+        test_coin_records = {
+            TEST_COIN_ID: TEST_COIN_RECORD,
+            TEST_COIN_ID2: TEST_COIN_RECORD2,
+        }
+        return test_coin_records.get(coin_id)
```

Comment:
this does look like it could be reused with the `get_coin_record` function in the other test in the other PR

Reply:
Done. I picked `get_coin_record_for_test_coins` as a name.